packages/react/src/SubdomainNavBar/NavigationVisbilityObserver.tsx

```diff
@@ -1,4 +1,4 @@
-import React, {useCallback, useRef, useState} from 'react'
+import React, {useCallback, useEffect, useRef, useState} from 'react'
```

Comment:
Extra useEffect here

Reply:
yeah, linters caught that too... already fixed

Reply:
Sweet!